model/riscv_sys_regs.sail

```diff
@@ -543,6 +543,57 @@ function legalize_mcountinhibit(c : Counterin, v : xlenbits) -> Counterin = {
   c
 }
 
+bitfield CountSmcntrpmf : bits(64) = {
+  MINH  : 62,
+  SINH  : 61,
+  UINH  : 60,
+  VSINH : 59,
+  VUINH : 58
+}
+register mcyclecfg : CountSmcntrpmf
+
+function should_inc_mcycle() -> bool = {
```

Comment:
I wonder if this should take a `priv` argument as well for consistency (since it's not immediately obvious if it's called before or after mode changes take effect).

Reply:
Agreed